addon/helpers/list.js

```diff
@@ -1,6 +1,16 @@
 import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
 
 export default MicroState.extend({
+
+  wrap(value) {
```

Comment:
Just noting for posterity that `wrap` is probably too generic, and does not convey intent. This is out of band for this PR since it would effect every microstate.

`toHandlebarsValue()` is probably a better name.